src/components/Countries.js

```diff
@@ -0,0 +1,104 @@
+import React, { useEffect } from 'react';
+import { useParams, Link } from 'react-router-dom';
+import { useSelector, useDispatch } from 'react-redux';
+import axios from 'axios';
+import { updateCountries, updateCountryData } from '../redux/actions/countriesActions';
+import flagsData from '../flagData';
+import '../styles/Countries.css';
+
+import continent1 from '../imagesContinents/continents/africa.png';
+import continent2 from '../imagesContinents/continents/america.png';
+import continent3 from '../imagesContinents/continents/asia.png';
+import continent4 from '../imagesContinents/continents/australia.png';
+import continent5 from '../imagesContinents/continents/europa.png';
+import continent6 from '../imagesContinents/continents/oceania.png';
+
+function getContinentImage(continent) {
+  switch (continent) {
+    case 'africa':
+      return continent1;
+    case 'america':
+      return continent2;
+    case 'asia':
```

Comment:
- [ ] I noticed one significant issue that should be addressed for the project to be more effective and functional 🙏 in this project,  we were suggested to use an API  that offers numeric data based on a parameter, with a preference for open or API key-protected options.

   However, The primary problem is the use of static data instead of a dynamic API, impacting real-time updates and user engagement 😨 Please fix this issue and use data from API instead of static 🙏 

    Transitioning to a dynamic API is recommended to meet project guidelines, ensure real-time updates, and offer a valuable learning opportunity for the developer

   **Remember to remove all static data and use only data from the API**